Flutter/signaturepad/how-to/custom-widget-on-flutterflow.md

```diff
@@ -0,0 +1,80 @@
+
+### Creating the custom widget
+
+1. Navigate to the `Custom Code` section in the left side navigation menu.
+2. Click on the `+ Add` button to open a dropdown menu. From there, select `Widget`.
+3. Update the widget name as desired.
+4. Click on the `View Boilerplate Code` button on the right side, represented by this icon `[</>]`.
+5. A popup will appear with startup code; locate the button labeled `</> Copy to Editor` and click on it.
+6. Save the widget.
+
+![Custom Widget](how-to-section-images/custom-widget.png)
+
+### Add SignaturePad widget as a dependency
+
+1. Click on `+ Add Dependency`, a text editor will appear.
+2. Navigate to [Syncfusion Flutter SignaturePad](https://pub.dev/packages/syncfusion_flutter_signaturepad) in [pub.dev](https://pub.dev/) and copy the dependency name and version using the `Copy to Clipboard` option.
+![Version](how-to-section-images/copy-version.png)
+3. Paste the copied dependency into the text editor, then click `Refresh` and `Save` it.
+
+>**Note**: The live version of [Syncfusion Flutter SignaturePad](https://pub.dev/packages/syncfusion_flutter_signaturepad) has been migrated to the latest version of Flutter SDK. To ensure compatibility, please check [FlutterFlow](https://app.flutterflow.io/dashboard)'s current Flutter version and obtain the corresponding version of [Syncfusion Flutter SignaturePad](https://pub.dev/packages/syncfusion_flutter_signaturepad) by referring to the [SDK compatibility](https://help.syncfusion.com/flutter/system-requirements#sdk-version-compatibility).
+
+![Dependency](how-to-section-images/dependency.png)
+
+>**Note**: If you are using an older version of a dependency instead of the latest one, remove the caret symbol (^) prefix in the version number after pasting the dependency. For example, change `^21.3.0` to `21.3.0`.
+>**Note**: Since [Syncfusion Flutter SignaturePad](https://pub.dev/packages/syncfusion_flutter_signaturepad) depends on the [Syncfusion Flutter Core](https://pub.dev/packages/syncfusion_flutter_core) package, make sure to add it as a dependency following the same steps mentioned above.
+
+### Import the package
+
+1. Navigate to the `Installing` tab on the [Syncfusion Flutter SignaturePad](https://pub.dev/packages/syncfusion_flutter_signaturepad) page. Under the `Import it` section, copy the package import statement.
+
+![Package](how-to-section-images/copy-package.png)
+
+2. Paste the copied import statement into the code editor and then `Save` it.
+
+![Import](how-to-section-images/import-package-flutterflow.png)
+
+### Add widget code snippet in code editor
+
+1. Navigate to the [Example](https://pub.dev/packages/syncfusion_flutter_signaturepad/example) tab in [Syncfusion Flutter SignaturePad](https://pub.dev/packages/syncfusion_flutter_signaturepad) and copy the widget specific codes.
+
+![Code](how-to-section-images/code-snippet.png)
+
+3. Paste the copied code snippet into the code editor, click `Format Code`, and `Save` it.
```

Comment:
3. Paste the copied code sample into the code editor, click `Format Code`, and `Save` it.